### ISSUE TITLE
Insufficient installation instruction 

### ISSUE THREAD

**Issue by ichbinsteffen** [zberkowitz/MASI#1]
Unfortunately by just following the instruction to download the HOALibrary and place it into the packages folder, MASI is not working.

    hoa.3d.scope~: no such object
    hoa.pi~: no such object
    hoa.3d.process~: no such object
    hoa.3d.encoder~: no such object

I'd like to get in touch, because I'd like to make use of MASI but your email adress is hidden in github.

**Comment by eliottparis** [comment]
@ichbinsteffen 
On which platform are you trying to install MASI / HOA ?
Everything is working fine for me on Mac following the instructions of the two repos.

1. Download the latest release of the *HoaLibrary* package for Max [here](https://github.com/CICM/HoaLibrary-Max/releases/tag/v2.2-beta) (depending on your platform), unzip it and put it in your Max Package directory.
2. Download the *MASI* package [here](https://github.com/zberkowitz/MASI/archive/master.zip), unzip it and put it in your Max Package directory.

**Comment by ichbinsteffen** [comment]
I tried it on windows. I'm not sure if I downloaded the wrong folder. 
I'm currently not on the windows machine, cant look it up right now. 
It seems to run fine on my macbook, but I've gotta make it work on windows for a windows only project. I'll come back later.
Is there something to know when trying to make it work on windows?

**Comment by eliottparis** [comment]
Yes, you should follow the same installation procedure for windows two (selecting the HoaLibrary-Win-v2.2.rar package from the release page).

**Comment by ichbinsteffen** [comment]
Okay, I did that and the HOA Library (opening hoa.overview) seems to work. 
But when I try to open MASI Max crashes. 
Edit:
Even Hoa Library brings max to crash. 
I tried to open the english tutorial patch for encoding and it crashed because of: 
hoa.pi.mxe64

**Comment by eliottparis** [comment]
Do you have any crash report ?

**Comment by ichbinsteffen** [comment]
Familiar with these kind of windows?
![unbenannt](https://user-images.githubusercontent.com/4134414/34113505-395b8eae-e410-11e7-9c2f-6e4740e581e9.PNG)

Full message: 
Problemsignatur: (Error signature)
  Problemereignisname:	APPCRASH (Error event name)
  Anwendungsname:	Max.exe (Application name)
  Anwendungsversion:	7.3.4.39163 (Application version)
  Anwendungszeitstempel:	5939ab65 (Application timestamp)
  Fehlermodulname:	hoa.3d.scope~.mxe64 (Error module name)
  Fehlermodulversion:	0.0.0.0 (Error module version)
  Fehlermodulzeitstempel:	559302cb (Error module timestamp)
  Ausnahmecode:	c000001d (Exception-Code)
  Ausnahmeoffset:	0000000000009b22 (Exception-Offset)
  Betriebsystemversion:	6.1.7601.2.1.0.256.48 (OS version)
  Gebietsschema-ID:	1031 (Region-ID)
  Zusatzinformation 1:	e808 (Additional info)
  Zusatzinformation 2:	e808284f81cfc66d5f961c526eedf787 (Additional info)
  Zusatzinformation 3:	f42f (Additional info)
  Zusatzinformation 4:	f42f0393c99fac575508d80d78889b36 (Additional info)

Does this tell you anything specific?

Edit: I'm going to have a look into the Windows Eventlog. Maybe it covers more detail.
Edit 2: No, nothing more.

**Comment by eliottparis** [comment]
[hoa.3d.scope~] seems to be the culprit... but this error message doesn't provide any specific informations about the error.
I've just tested HoaLibrary on Windows 10 and everything is working fine for me...

What is your OS version ?
If you see the HoaLibrary message in the Max Console when Max is loaded, the HoaLibrary is correctly installed.
What happens when you simply instantiate a [hoa.pi] object in a blank patcher ? same for hoa.3d.scope~ ? 
I need more information to reproduce your error !

**Comment by ichbinsteffen** [comment]
> 6.1.7601.2.1.0.256.48 (OS version)

This is the exact windows version. In other words Windows 7 Professional 64Bit. 
When I start Max, the console shows no messages at all. 
I've put the unpacked Hoa Lib folder into: C:\Users\admin\Documents\Max 7\Packages 
(yes, admin is the account I use).
And when I want to start Hoa Lib, I click (in the menu-bar) "Extras > Hoa Library > hoa.overview" 
When I'm at home I'll try to reproduce this error on my private machine using Windows 10. I'll let you know what happened.

**Comment by ichbinsteffen** [comment]
It works on my private computer at home using windows 10. 
It could be that it doesn't work with windows 7, I have no Idea why though. 
Anyhting else I could try to confirm this?

**Comment by ichbinsteffen** [comment]
Well unfortunately I gave it up to make it run on windows 7. 
Meanwhile I've created my own OSC server around Ambisonics Toolkit. 
Thank you for your help though :)
